v2.11.0/language-tr.yml

```diff
@@ -0,0 +1,338 @@
+- "Tavşanın görünümünü değiştirir"
+msg-button-collar-color: "&aRengi değiştir"
+msg-button-collar-color-lore:
+- "Varlığın tasmasının rengini değiştirir"
+msg-button-wolf-angry: "&aDurumu değiştir"
+msg-button-wolf-angry-lore:
+- "Kurtun kızgın halini değiştirir"
+msg-button-carrying-chest: "&aDurumu değiştir"
+msg-button-carrying-chest-lore:
+- "Varlığın sandıklı halini değiştirir"
+msg-button-horse-color: "&aRengi değiştir"
+msg-button-horse-color-lore:
+- "Atın rengini değiştirir"
+msg-button-horse-style: "&aÇeşiti değiştir"
+msg-button-horse-style-lore:
+- "Atın görünümünü değiştirir"
+msg-button-horse-armor: "&aZırhı değiştir"
+msg-button-horse-armor-lore:
+- "Atın zırhını değiştirir"
+msg-button-llama-variant: "&aÇeşiti değiştir"
+msg-button-llama-variant-lore:
+- "Lamanın görünümünü değiştirir"
+msg-button-llama-carpet-color: "&aRengi değiştir"
+msg-button-llama-carpet-color-lore:
+- "Lamanın halı rengini değiştirir"
+msg-button-creeper-charged: "&aDurumu değiştir"
+msg-button-creeper-charged-lore:
+- "Creeperın şarjlı halini değiştirir"
+msg-button-fox-variant: "&aÇeşiti değiştir"
+msg-button-fox-variant-lore:
+- "Tilkinin görünümünü değiştirir"
+msg-button-fox-crouching: "&aPozisyonu değiştir"
+msg-button-fox-crouching-lore:
+- "Tilkinin oturma halini değiştirir"
+msg-button-fox-sleeping: "&aPozisyonu değiştir"
+msg-button-fox-sleeping-lore:
+- "Tilkinin uyuma halini değiştirir"
+msg-button-mooshroom-variant: "&aÇeşiti değiştir"
+msg-button-mooshroom-variant-lore:
+- "Mantar inek görünümünü değiştirir"
+msg-button-panda-variant: "&aÇeşiti değiştir"
+msg-button-panda-variant-lore:
+- "Pandanın görünümünü değiştirir"
+msg-button-parrot-variant: "&aÇeşiti değiştir"
+msg-button-parrot-variant-lore:
+- "Papağanın görünümünü değiştirir"
+msg-button-pig-saddle: "&aDurumu değiştir"
+msg-button-pig-saddle-lore:
+- "Domuzun eğerli halini değiştirir"
+msg-button-sheep-color: "&aRengi değiştir"
+msg-button-sheep-color-lore:
+- "Koyunun yün rengini değiştirir"
+msg-button-sheep-sheared: "&aDurumu değiştir"
+msg-button-sheep-sheared-lore:
+- "Koyunun traşlı halini değiştirir"
+msg-button-villager-profession: "&aMesleği değiştir"
+msg-button-villager-profession-lore:
+- "Köylünün mesleğini ve görünümünü değiştirir"
+msg-button-villager-variant: "&aÇeşiti değiştir"
+msg-button-villager-variant-lore:
+- "Köylünün biyomunu ve görünüşünü değiştirir"
+msg-button-villager-level: "&aRozeti değiştir"
+msg-button-villager-level-lore:
+- "Köylünün seviyesini gösteren rozeti değiştirir"
+msg-button-zombie-villager-profession: "&aMesleği değiştir"
+msg-button-zombie-villager-profession-lore:
+- "Zombi köylünün mesleğini ve görünümünü değiştirir"
+msg-button-slime-size: "&aBoyutu değiştir"
+msg-button-slime-size-lore:
+- "Slime boyutunu değiştirir."
+- "Şuanki boyut: &e{size}"
+msg-button-magma-cube-size: "&aBoyutu değiştir"
+msg-button-magma-cube-size-lore:
+- "Magma küpü boyutunu değiştirir."
+- "Şuanki boyut: &e{size}"
+
+msg-trading-title-prefix: "&a"

```

Comment:
The default uses color code `&2` here, so this should as well for consistency reasons.